auth-web/src/components/auth/manage-business/AffiliationAction.vue

```diff
@@ -153,6 +153,8 @@ export default defineComponent({
         if (amalgamationTypes.includes(business.nameRequest?.legalType)) {
           payload = { filingType: FilingTypes.AMALGAMATION_APPLICATION, business }
         }
+      } else if (business.nameRequest?.requestActionCd === NrRequestActionCodes.MOVE) {
+        payload = { filingType: FilingTypes.CONTINUATION_IN, business }
```

Comment:
Please add a check here for the feature flag. Should be something similar to what amalgamations do (line 138 and line 153).

Reply:
Fixed, thanks! Resolving.

Reply:
Looks great, Kevin! 👍